Source/cmake/OptionsWPE.cmake

```diff
@@ -175,7 +176,7 @@ endif ()
 if (WPE_API_VERSION VERSION_EQUAL "1.0")
     CALCULATE_LIBRARY_VERSIONS_FROM_LIBTOOL_TRIPLE(WEBKIT 21 0 18)
 else ()
-    CALCULATE_LIBRARY_VERSIONS_FROM_LIBTOOL_TRIPLE(WEBKIT 2 0 2)
+    CALCULATE_LIBRARY_VERSIONS_FROM_LIBTOOL_TRIPLE(WEBKIT 3 0 3)
```

Comment:
New api version is incompatible change so we should do C + 1, 0, 0. But still, I think it should be safer to dd a new api version instead of changing existing and released 1.1 one.

Reply:
Ah, you're right. I assumed I knew how to use the libtool versions without looking it up... shame on me.

Anyway, since this patch changes the name of the API, the generated library name will be totally different, so I can simply reset it down to `0 0 0`, OK?

Reply:
> Anyway, since this patch changes the name of the API, the generated library name will be totally different, so I can simply reset it down to 0 0 0, OK?

Will do this.